wallet/wallet.go

```diff
@@ -1848,6 +1848,36 @@ func (w *Wallet) MasterPubKey(account uint32) (*hdkeychain.ExtendedKey, error) {
 	return extKey, nil
 }
 
+// CoinTypeKey returns the BIP0044 coin type private key for the passed account.
```

Comment:
This comment is incorrect.  There is no passed account.

Please name this CoinTypePrivKey to match the udb method.